src/controllers/Event_hostsController.php

```diff
@@ -141,7 +141,7 @@ public function removeHostFromEvent(Request $request, PDO $db)
 
         $mapper = $this->getEventHostMapper($request, $db);
 
-        $uid = $mapper->removeHostFromEvent($user_id, $event_id);
+        $uid = $mapper->removeHostfromEvent($user_id, $event_id);
```

Comment:
See [`EventHostMapper::removeHostfromEvent()`](https://github.com/joindin/joindin-api/blob/0e120fc5ea508e9483a926a1c82c91cb928f56ec/src/models/EventHostMapper.php#L59-L69).

Alternatively, of course, we can rename the method (which in my opinion makes more sense),